app/threads/client/threads.css

```diff
@@ -78,11 +100,13 @@
 		}
 	}
 
-	& > .thread-quote {
-
+	& .thread-quote {
 		display: flex;
 
-		margin: calc((var(--default-padding) /2) - 2px) 0 2px 0;
+		margin: 0 -24px 8px -24px;
```

Comment:
Codacy found an issue: [Unexpected longhand value '0 -24px 8px -24px' instead of '0 -24px 8px' (shorthand-property-no-redundant-values)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5416119)